File: 0070-climbing-stairs/0070-climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
        if (n <= 3) return n;
        int prev = 2;
        int cur = 3;
        for (int i = 3; i < n; i++){
            int tmp = cur;
            cur = cur + prev;
            prev = tmp;
        }
        return cur;
    }
}

File: 0070-climbing-stairs/0070-climbing-stairs.cs
public class Solution {
    public int ClimbStairs(int n) {
        if (n <= 3) return n;
        int a = 1, b = 1;
        for (int i = 0; i < n; i++){
            int tmp = b;
            b = a;
            a+= tmp;
        }
        return b;
    }
}